packages/extension-vscode/src/server.ts

```diff
@@ -17,7 +21,31 @@ connection.onInitialize((params) => {
      */
     workspace = params.rootPath || '';
 
-    return { capabilities: { textDocumentSync: TextDocumentSyncKind.Full } };
+    const resultObject: InitializeResult = {capabilities: {textDocumentSync: TextDocumentSyncKind.Full}};
+
+    resultObject.capabilities.codeActionProvider = true;
+    resultObject.capabilities.executeCommandProvider = {
+        commands: [
+            'vscode-webhint/ignore-hint-project',
+            'vscode-webhint/ignore-hint-global',
+            'vscode-webhint/ignore-problem-project',
+            'vscode-webhint/ignore-problem-global',
+            'vscode-webhint/edit-hintrc-project',
+            'vscode-webhint/edit-hintrc-global'
+        ]
+    };
+
+    return resultObject;
+});
+
+connection.onNotification(notifications.reloadAllProjectsConfig, async () => {
+    const message = `Reloading .hintrc file from:\n ${globalStoragePath}`;
+
+    console.log(message);
+    analyzer.onConfigurationChanged();
+    await Promise.all(documents.all().map((doc) => {
+        return analyzer.validateTextDocument(doc, globalStoragePath);
```

Comment:
Passing `globalStoragePath` here forces loading webhint itself from the shared location (not just using the global config). We still want to use the project's version of webhint if one is present.